### PR TITLE
Death Screen + Player Input Rework

### PR DESCRIPTION
**Pull request by ynks**

### CODE REVIEW

**Comment by ynks** [comment]
https://github.com/user-attachments/assets/50f44180-7348-4154-991f-ed9a80cad2c4

**Comment by dariormt** [comment]
let-him-cook-cook

**Comment by ynks** [comment]
@dariormt do pull review (do it seriously), also we have to decide if this is merged to Dev or to Prototype-Scene

File: Assets/Enemy/EnemyDamageable.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class EnemyDamageable : Damageable
+public class  EnemyDamageable : Damageable
```
Comment (dariormt) [inline Assets/Enemy/EnemyDamageable.cs:5]: Extra space here

File: Assets/Player/CS_PlayerHealth.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
+using JetBrains.Annotations;
```
Comment (dariormt) [inline Assets/Player/CS_PlayerHealth.cs:1]: For what are you using Jet Brains Annotations?

**Review by dariormt — APPROVED** [review]
Looks good, just some minor, non-important changes :)

File: Assets/Player/CS_PlayerHealth.cs (hunk 2 above)
Comment (ynks) [reply Assets/Player/CS_PlayerHealth.cs:1]: Tbh I dont really know what they are but its not grayed out so i didnt delete it, will do in class today and see what happens, we def not want unused dependencies

**Review by dariormt — APPROVED** [review]

**Pull request merged**